### PR TITLE
reference/tools: change binlog name in DM docs

### PR DESCRIPTION
**Pull request by yikeke**
### What is changed, added or deleted? 

change binlog name in DM docs

### What is the related PR or file link(s)? 

To align with: https://github.com/pingcap/docs-cn/pull/1737
Ref: https://github.com/pingcap/docs-cn/pull/1679

### Which version does your change affect? 

3.0, 2.1, dev

### CODE REVIEW

File: dev/reference/tools/data-migration/configure/task-configuration-file.md [hunk 1]
```diff
@@ -148,7 +148,7 @@ mysql-instances:
     source-id: "mysql-replica-01"                                      # The ID of the upstream instance or replication group ID. It can be configured by referring to the `source_id` in the `inventory.ini` file or the `source-id` in the `dm-master.toml` file.
     meta:                                                              # The position where the binlog replication starts when `task-mode` is `incremental` and the downstream database checkpoint does not exist. If the checkpoint exists, the checkpoint is used.
 
-      binlog-name: binlog-00001
+      binlog-name: binlog-000001
```
Comment (lilin90) [inline dev/reference/tools/data-migration/configure/task-configuration-file.md]:
```suggestion
      binlog-name: binlog.000001
```

File: v2.1/reference/tools/data-migration/configure/task-configuration-file.md [hunk 2]
```diff
@@ -148,7 +148,7 @@ mysql-instances:
     source-id: "mysql-replica-01"                                      # The ID of the upstream instance or replication group ID. It can be configured by referring to the `source_id` in the `inventory.ini` file or the `source-id` in the `dm-master.toml` file.
     meta:                                                              # The position where the binlog replication starts when `task-mode` is `incremental` and the downstream database checkpoint does not exist. If the checkpoint exists, the checkpoint is used.
 
-      binlog-name: binlog-00001
+      binlog-name: binlog-000001
```
Comment (lilin90) [inline v2.1/reference/tools/data-migration/configure/task-configuration-file.md]:
```suggestion
      binlog-name: binlog.000001
```

File: v3.0/reference/tools/data-migration/configure/task-configuration-file.md [hunk 3]
```diff
@@ -149,7 +149,7 @@ mysql-instances:
     source-id: "mysql-replica-01"                                      # The ID of the upstream instance or replication group ID. It can be configured by referring to the `source_id` in the `inventory.ini` file or the `source-id` in the `dm-master.toml` file.
     meta:                                                              # The position where the binlog replication starts when `task-mode` is `incremental` and the downstream database checkpoint does not exist. If the checkpoint exists, the checkpoint is used.
 
-      binlog-name: binlog-00001
+      binlog-name: binlog-000001
```
Comment (lilin90) [inline v3.0/reference/tools/data-migration/configure/task-configuration-file.md]:
```suggestion
      binlog-name: binlog.000001
```

**Comment by lilin90** [comment]
@yikeke Is this PR also related to https://github.com/pingcap/docs-cn/pull/1679?

**Comment by yikeke** [comment]
> @yikeke Is this PR also related to [pingcap/docs-cn#1679](https://github.com/pingcap/docs-cn/pull/1679)?

No, but #1679 explains why I didn't align this PR with other file changes in #1737.

**Comment by lilin90** [comment]
> > @yikeke Is this PR also related to [pingcap/docs-cn#1679](https://github.com/pingcap/docs-cn/pull/1679)?
> 
> I think #1679 explains why I didn't align this PR with other file changes in #1737.

Got it.

**Pull request merged**